Util/Transliterator.php

```diff
@@ -7,8 +7,9 @@ class Transliterator
     public static function transliterate($string)
     {
         // needs intl extension
-        if (function_exists('transliterator_transliterate')) {
-            $string = transliterator_transliterate("Any-Latin; Latin-ASCII; [\u0100-\u7fff] remove" , $string);
+        $transId = "Any-Latin; Latin-ASCII; [\u0100-\u7fff] remove";
+        if (function_exists('transliterator_transliterate') && $transliterator = \Transliterator::create($transId)) {
```

Comment:
Instead of checking the existence of the `transliterator_transliterate` function, could you check that the `\Transliterator` exists?